cmd/zetaclientd/debug.go

```diff
@@ -97,10 +97,10 @@ func DebugCmd() *cobra.Command {
 
 			if chains.IsEVMChain(chain.ChainId) {
 
-				ob := evm.ChainClient{
+				ob := evm.Client{
```

Comment:
Also think we should use `observer` instead of `ob`. I think clarity is a good tradeoff for verbosity

Reply:
same commit  https://github.com/zeta-chain/node/pull/2150/commits/6a2a68a8c776e333d5bad07894ab4962fba64d17